src/EditorFeatures/Core/GoToDefinition/GoToDefinitionContext.cs

```diff
@@ -24,19 +25,20 @@ public GoToDefinitionContext(Document document, int position, CancellationToken
         public int Position { get; }
         public CancellationToken CancellationToken { get; }
 
-        public IReadOnlyDictionary<string, List<DefinitionItem>> Items => _items;
+        public IReadOnlyDictionary<string, List<DefinitionItem>> Items => null;// _items;
         public TextSpan Span { get; set; }
 
+        public bool TryGetItems(string key, out IEnumerable<DefinitionItem> items)
```

Comment:
I didn't want to directly expose the underlying multidictionary, hence this public member.